backend/.gitignore

```diff
@@ -1 +1,3 @@
 __pycache__
+build/
+src/backend.egg-info/
```

Comment:
could we change this to `*.egg-info`? (because that is the format build artifacts use)